frontend/packages/dev-console/src/components/PodsRing.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import { VictoryLabel } from 'victory';
+import * as _ from 'lodash';
+import { K8sResourceKind, k8sPatch, K8sKind } from '@console/internal/module/k8s';
+import { AngleUpIcon, AngleDownIcon } from '@patternfly/react-icons';
+import PodStatus from './topology/shapes/PodStatus';
+import { Pod } from './topology/topology-types';
+import SvgDefsProvider from './svg/SvgDefsProvider';
+
+interface PodsRingProps {
+  pods: Pod[];
+  obj: K8sResourceKind;
+  resourceKind: K8sKind;
+  path: string;
+  ringContainerSize: number;
```

Comment:
Probably unnecessary for now as there's no need for any component to render at a different size.